### PR TITLE
`dart fix --apply` from Flutter 3.21.0-1.0.pre.2

### PR DESCRIPTION
**Pull request by domesticmouse**
Hey @QuncCccccc, any views on the fixes suggested by `dart fix`?

This PR needs to be held for the next stable update.

## Pre-launch Checklist

- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I read the [Contributors Guide].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-devrel channel on [Discord].

[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[Contributors Guide]: https://github.com/flutter/samples/blob/main/CONTRIBUTING.md

### CODE REVIEW

File: material_3_demo/lib/color_palettes_screen.dart [hunk 1]
```diff
@@ -303,13 +303,13 @@ class ColorSchemeView extends StatelessWidget {
           children: [
             ColorChip(
               label: 'background',
```
Comment (QuncCccccc) [inline material_3_demo/lib/color_palettes_screen.dart:305]: We can actually remove these chips whose colors have been deprecated:)

File: material_3_demo/lib/color_palettes_screen.dart [hunk 2]
```diff
@@ -251,13 +251,13 @@ class ColorSchemeView extends StatelessWidget {
             ),
             ColorChip(
               label: 'surfaceVariant',
```
Comment (QuncCccccc) [inline material_3_demo/lib/color_palettes_screen.dart:253]: Also here, I think we can remove this:)

**Review by QuncCccccc — APPROVED** [review]
Sorry for the late response. I was OOO last week. LGTM! For the deprecated colors, I think we can just remove the corresponding `ColorChip`s. Leaving them here for now is also okay because I will update this ColorScheme page soon to add all the new colors and remove the deprecated colors (If they are still here:)).

File: material_3_demo/lib/color_palettes_screen.dart (hunk 1 above)
Comment (domesticmouse) [reply material_3_demo/lib/color_palettes_screen.dart:305]: Feel free to adopt this change and adapt it to suit your needs. I was just getting in early to see what the required changes for the next stable release might look like.

Comment (QuncCccccc) [reply material_3_demo/lib/color_palettes_screen.dart]: Just pushed a small change to remove the deprecated color chips! :)

**Comment by domesticmouse** [comment]
Replaced by #2266

**Pull request closed without merging**